### ISSUE TITLE
Upgrading bundler breaks cache

### ISSUE THREAD

**Issue by JustinAiken** [Shopify/bootsnap#46]
After uninstalling `bundler` 1.14.6 and installing 1.15.0, got this the first time I tried to do something:

```
/Users/jaiken/.rvm/gems/ruby-2.3.4/gems/bootsnap-0.2.14/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:17:in `require': cannot load such file -- /Users/jaiken/.rvm/gems/ruby-2.3.4/gems/bundler-1.14.6/lib/bundler.rb (LoadError)
	from /Users/jaiken/.rvm/gems/ruby-2.3.4/gems/bootsnap-0.2.14/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:17:in `require'
	from /Users/jaiken/.rvm/gems/ruby-2.3.4/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:274:in `block in require'
	from /Users/jaiken/.rvm/gems/ruby-2.3.4/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /Users/jaiken/.rvm/gems/ruby-2.3.4/gems/activesupport-4.2.8/lib/active_support/dependencies.rb:274:in `require'
	from /Users/jaiken/.rvm/gems/ruby-2.3.4/gems/bundler-1.15.0/lib/bundler/shared_helpers.rb:263:in `prints_major_deprecations?'
	from /Users/jaiken/.rvm/gems/ruby-2.3.4/gems/bundler-1.15.0/lib/bundler/shared_helpers.rb:133:in `major_deprecation'
	from /Users/jaiken/.rvm/gems/ruby-2.3.4/gems/bundler-1.15.0/lib/bundler/rubygems_integration.rb:438:in `block in replace_bin_path'
	from /Users/jaiken/.rvm/gems/ruby-2.3.4/gems/bundler-1.15.0/lib/bundler/rubygems_integration.rb:463:in `block in replace_bin_path'
```

**Comment by burke** [comment]
Fixed in either 1.2.0 or 1.3.0, sorry for the lack of response.

**Comment by JustinAiken** [comment]
Thanks! Now that you mention, haven't noticed bootsnap at all for the last few months other than faster startup times! 😀
